rocky/tools/management/commands/setup_dev_account.py

```diff
@@ -45,6 +45,7 @@ def handle(self, *args, **options):
                 "view_organizationmember",
                 "add_organizationmember",
                 "change_organization",
+                "can_scan_organization",
```

Comment:
We need to create a migration to also add this permission to the admin group in existing installations.